src/libsystemd-network/test-ipv4ll-manual.c

```diff
@@ -57,6 +57,9 @@ static int client_run(int ifindex, const char *seed_str, const struct ether_addr
                 assert_se(sd_ipv4ll_set_address_seed(ll, seed) >= 0);
         }
 
+        if (start_address && start_address->s_addr != 0)
+                assert_se(sd_ipv4ll_set_address(ll, start_address) >= 0);
```

Comment:
I'd like `sd_ipv4ll_set_address()` also handle `NULL` for the address gracefully.

Reply:
```
        if (in4_addr_is_link_local(start_address))
                assert_se(sd_ipv4ll_set_address(ll, start_address) >= 0);
```
If I handle `NULL` with the `in4_addr_is_link_local` this assertion occures.
```
$ ./test-ipv4ll-manual wlan0
Assertion 'a' failed at src/basic/in-addr-util.c:47, function in4_addr_is_link_local(). Aborting.
Abgebrochen
```
And if I handle 'NULL` with the `sd_ipv4ll_set_address` this assertion occures.
```
$ ./test-ipv4ll-manual wlan0
Assertion 'address' failed at src/libsystemd-network/sd-ipv4ll.c:228, function sd_ipv4ll_set_address(). Ignoring.
Assertion 'sd_ipv4ll_set_address(ll, start_address) >= 0' failed at src/libsystemd-network/test-ipv4ll-manual.c:61, function client_run(). Aborting.
Abgebrochen
```

Reply:
Hm, OK. Still I'd like to make `sd_ipv4ll_set_address()` and `sd_ipv4acd_set_address()` accept NULL. But that's out of scope of this PR. Please ignore that.